packages/app-store/_utils/updateAppServerLocation.ts

```diff
@@ -0,0 +1,15 @@
+import prisma from "@calcom/prisma";
+
+import { appKeysSchema as zohoKeysSchema } from "../zohocalendar/zod";
+
+async function updateAppServerLocation(slug: string, serverUrl: string) {
+  const app = await prisma.app.findUnique({ where: { slug } });
+  const { client_id, client_secret } = zohoKeysSchema.parse(app?.keys) || {};
+  const updatedKeys = { client_id, client_secret, server_location: serverUrl };
+  await prisma.app.update({
```

Comment:
We shouldn't write the server location to the app data as this will affect all users. We should be writing the location as a part of the user's Zoho credentials